Units/function-parameters.r/python-fp-test.d/expected.tags

```diff
@@ -0,0 +1,3 @@
+fun	input.py	/^def fun():$/;"	f
```

Comment:
ctags never runs Perl parser ( and FunctionParameter subparser ) for .py files.
So this test case doesn't make sense as a test case for FunctionParameter subparser.
This test case tests the main part of ctags.
Please remove this test case.